backends/jms/pom.xml

```diff
@@ -41,6 +41,12 @@
             <type>test-jar</type>
             <scope>test</scope>
         </dependency>
+
+        <!-- Avoid warnings during build -->
+        <dependency>
```

Comment:
ok but is "provided" correct? That maven definition always confuses me, but we don't really expect it to be provided at runtime.